core/log-parser/index.js

```diff
@@ -14,7 +14,12 @@ export default class LogParser extends EventEmitter {
 
     options.filename = filename;
 
-    this.eventStore = {};
+    this.eventStore = {
+      disconnected: {}, // holding area, cleared on map change.
+      players: {}, // persistent data, steamid, controller, suffix.
+      session: {}, // old eventstore, nonpersistent data
```

Comment:
At some point, we should split these out to separate objects as well.

Not critical for this PR though.

Reply:
This PR is heavily cutdown version of a log parser change I tested with BB | wasp, where on startup squadjs scans the entire log file and builds up these caches/syncs state. There are issues around that PR and how it was built, but 
I'm testing a version where the eventstore actually ends up as a cache in the server object. 

As is, currently no plugins really go back and touch this information, and our information in server.players gets refreshed every cycle; but we are not "attaching" the log specific entries here over there. 

In my branch, I'm using the logparser cache to do lookups by player controller directly in damage events and so on.

We also end up with a way of tracking sessions directly on each player, but thats also possible elsewhere.

Reply:
It's not critical for this PR, so this can be resolved. However, my chain of thoughts is currently everything in that object is keyed by just a player's name, so if we start adding more events that overlap then data could be overwritten unintentionally. If all the data in this object just related to damage/wound/death/revive that's fine, we just need to apply a more useful name for the object.